sdk/cosmos/azure-cosmos/test/test_latest_session_token.py

```diff
@@ -0,0 +1,214 @@
+                                                                                    previous_session_token,
+                                                                                    feed_ranges_and_session_tokens)
+        session_token = container.get_latest_session_token(feed_ranges_and_session_tokens, target_feed_range)
+
+        assert session_token == target_session_token
+        feed_ranges_and_session_tokens.append((target_feed_range, session_token))
+
+        self.trigger_split(container, 11000)
+
+        target_session_token, _ = self.create_items_logical_pk(container, target_feed_range, session_token,
+                                                               feed_ranges_and_session_tokens)
+        target_feed_range = container.feed_range_from_partition_key(target_pk)
+        session_token = container.get_latest_session_token(feed_ranges_and_session_tokens, target_feed_range)
+
+        assert session_token == target_session_token
+        self.database.delete_container(container.id)
+
+    def test_latest_session_token_from_physical_pk(self):
+        container = self.database.create_container("test_updated_session_token_from_physical_pk" + str(uuid.uuid4()),
+                                                   PartitionKey(path="/pk"),
+                                                    offer_throughput=400)
+        feed_ranges_and_session_tokens = []
+        previous_session_token = ""
+        pk_feed_range = container.feed_range_from_partition_key('A1')
+        target_session_token, target_feed_range, previous_session_token = self.create_items_physical_pk(container, pk_feed_range,
+                                                                                                   previous_session_token,
+                                                                                                   feed_ranges_and_session_tokens)
+
+        session_token = container.get_latest_session_token(feed_ranges_and_session_tokens, target_feed_range)
+        assert session_token == target_session_token
+
+        self.trigger_split(container, 11000)
+
+        _, target_feed_range, previous_session_token = self.create_items_physical_pk(container, pk_feed_range,
+                                                                                session_token,
+                                                                                feed_ranges_and_session_tokens)
+
+        session_token = container.get_latest_session_token(feed_ranges_and_session_tokens, target_feed_range)
+        assert is_compound_session_token(session_token)
+        session_tokens = session_token.split(",")
+        assert len(session_tokens) == 2
+        pk_range_id1, session_token1 = parse_session_token(session_tokens[0])
+        pk_range_id2, session_token2 = parse_session_token(session_tokens[1])
+        pk_range_ids = [pk_range_id1, pk_range_id2]
+
+        assert 320 >= (session_token1.global_lsn + session_token2.global_lsn)
```

Comment:
What does 320 mean here?

Reply:
Should have been ==, but I am just checking that it is the correct session tokens by comparing glsn to number of changes.